### PR TITLE
feat(analytics): generate weekly value reports

### LINKED ISSUE #138: feat(analytics): add weekly Gittensory value report
## Background

If the goal is higher Gittensor allocation, Gittensory needs a concise value report showing adoption, maintainer burden reduction, and signal freshness.

## Goal

Generate an operator report suitable for explaining Gittensory’s value to the Gittensor team.

## Current Behavior

Usage and impact have to be inferred manually from scattered data.

## Desired Behavior

Gittensory can produce a weekly value report with public-safe and private operator variants.

## Implementation Requirements

- Summarize active users, active repos, MCP usage, GitHub commands, PR packets, quiet skips, drift detections, and maintainer-value signals.
- Separate public-safe aggregate output from private operator details.
- Include freshness and data-quality warnings.
- Support scheduled generation.
- Keep output free of private source and identity leaks.

## Public/Private Output Boundaries

Public GitHub output must never include wallets, hotkeys, payout/reward estimates, raw trust scores, public score estimates, private reviewability, private scoreability context, or farming language. Private API/MCP/control-panel surfaces may show authenticated scoreability, blockers, projections, and evidence, but must not claim guaranteed payout outcomes.

## Acceptance Criteria

- Shows adoption, usage, quiet skips, PRs preflighted, packets generated, drift detected, and maintainer value signals.
- Public-safe version excludes private details.
- Private operator version includes deeper metrics.
- Tests cover redaction and stale-data warnings.

## Testing Requirements

- `npm run test:ci` must pass.
- Global coverage must remain at or above 97% for lines, statements, functions, and branches.
- Aim for 98%+ branch coverage locally.
- Add tests for every new branch, fallback path, sanitizer rule, and regression.
- Add invariant/property-style tests when behavior depends on sorting, gating, scoring, source-upload safety, public/private boundaries, or telemetry privacy.
- MCP/local tooling must prove source contents are not uploaded when local metadata is involved.
- Public GitHub output must be tested against forbidden language: wallet, hotkey, raw trust score, payout, reward estimate, farming, private reviewability, and public score estimate.

### PR DESCRIPTION
**Pull request by oktofeesh1**
## Summary
- add a rollup-backed weekly value report for public-safe and operator views
- wire the report into the operator dashboard, app API, internal job queue, and Monday scheduled generation
- add privacy/auth regression coverage for public-safe output, internal guards, queue processing, and scheduler behavior

## What changed
- builds weekly adoption, MCP usage, GitHub command, PR packet, quiet-skip, drift, and maintainer-value metrics from daily product usage rollups
- keeps public reports aggregate-only and genericizes public warning text while preserving detailed warning context for operator reports
- exposes an authenticated app report endpoint and protected internal queue/run endpoints
- updates the UI OpenAPI artifact and operator dashboard card to display report freshness and warnings

## Why
Weekly operator text was static and could not reflect product usage or maintainer value. This gives operators a generated report while keeping public-safe variants free of source, token, local-path, wallet, trust, payout, and private scoring leakage.

## Validation
- `npm run test:unit -- test/unit/weekly-value-report.test.ts`
- `npm run test:integration -- test/integration/api.test.ts test/integration/routes-errors.test.ts`
- `npm run test:ci`
- Codex Security diff scan: no reportable findings; 13/13 worklist rows completed; report format validated and rendered locally.

## Notes
- Stacked after the analytics usage-event spine and daily-rollup branches; the weekly report delta is commit `2046ed8`.
- Closes #138.
- Advances #134.

### CODE REVIEW

File: src/api/routes.ts [hunk 1]
```diff
@@ -140,10 +146,55 @@ import { buildLocalBranchAnalysis, findCurrentBranchPullRequest } from "../signa
...
+  ProductUsageOutcome,
+  ProductUsageSurface,
+  RepoSyncSegmentRecord,
+  RepositoryRecord,
+} from "../types";
 import { errorMessage, nowIso } from "../utils/json";
 
 type AppBindings = { Bindings: Env };
+type AppContext = Context<AppBindings>;
+
```
[the remark this answers is not in the document]
Comment (oktofeesh1) [reply src/api/routes.ts:172]: Fixed in the latest head. The standalone weekly value report endpoint now defaults to the public-safe variant, and the operator report is only served for an explicit `variant=operator` request behind the operator role check. Added regression coverage for the public default, owner access to public-safe output, owner rejection from operator output, and continued redaction of sensitive report content. Fresh Superagent Security Scan is passing on this head.

**Review by JSONbored — APPROVED** [review]
@oktofeesh1 This is ready from my side. Weekly value reports are green and merge-clean against current `main`.

A few notes:
- The report service has focused unit coverage.
- The API, OpenAPI, analytics UI, and operator UI surfaces are wired together.
- Empty-data and scheduled queue paths are covered, which is the right shape for this feature.
